packages/ceramic-core/src/anchor/anchor-service-response.ts

```diff
@@ -0,0 +1,15 @@
+import CID from 'cids'
+
+export default class AnchorServiceResponse {
```

Comment:
Good work on the well-defined request/response models

Reply:
👍